src/routes/order.js

```diff
@@ -1,14 +1,129 @@
-const express = require('express');
+  const express = require('express');
 
-const router = express.Router();
+ const router = express.Router();
 
 const orderController = require('../controllers/orderController');
 const { validOrder } = require('../validation/validator');
+/**
+ * @swagger
+ * tags:
+ *   name: Order
+ *   description: API endpoints for Order
+ */
+/**
+ * @swagger
+ * /api/order/:
+ *   get:
+ *     tags: [Order]
+ *     summary: Get all Order
+ *     responses:
+ *       201:
+ *         description: Successful operation
+ *       400:
+ *         description: Internal server error
+ */
 
-router.get('/', orderController.getAllOrders);
+ router.get('/', orderController.getAllOrders);
+
+/**
+ * @swagger
+ * /api//Order/:
+ *   post:
+ *     tags: [Order]
+ *     summary: Add a new Order
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/Order_Add'
```

Comment:
Order_Add should be Schema in swagger.js file. I cannot see this schema in theswagger file.